src/content/getting-started/install-an-application/_index.md

```diff
@@ -15,4 +15,238 @@ user_questions:
+
+```nohighlight
+kubectl config set-context --current --namespace=org-namespace
+```
+
+That way you ensure that the App CR is created in the correct namespace.
+
+## Step 1: Check applications installed in the cluster
+
+First we will take a look to the existing applications already running in the cluster. By default the cluster has some default apps to make the cluster operational. Since you can have more than one cluster in the namespace, let's filter the apps by the cluster name.
+
+Ensure you are logged into the management cluster.
+
+```nohighlight
+$ kubectl gs -n org-namespace get apps
+NAME                                 VERSION          CREATED_AT   LAST_DEPLOYED        STATUS
+test01                                 2.0.0               13m          12m             deployed
+test01-app-operator                    6.11.0              12m          12m             deployed
+test01-aws-ebs-csi-driver-smons        0.1.0               12m          42s             deployed
+test01-aws-pod-identity-webhook        1.16.0              12m          117s            deployed
+...
+test01-observability-bundle            1.5.2               12m          4m28s           deployed
+test01-security-bundle                 1.8.0               12m          3m32s           deployed
+test01-teleport-kube-agent             0.9.2               12m          4m24s           deployed
+test01-vertical-pod-autoscaler         5.2.4               12m          110s            deployed
+```
+
+__Note__: We don't enforce the cluster prefix (here: `test01-`), but it's a good practice to have it.
+
+You can see that several applications already exist for the workload cluster `test01`. Most of the apps run directly in the workload cluster itself. The operator which actually deploys the applications to the workload clusters is running on the management cluster and is called `app-operator`. Learn more about this process [in this guide]({{< relref "/vintage/platform-overview/app-platform/" >}}).
+
+## Step 2: Install an ingress nginx controller
+
+Before we install our publicly accessible `hello-world` app, we need to have an ingress controller running in the cluster. The ingress controller is responsible for routing the incoming traffic to the correct service in the cluster and make it available to the public.
+
+To know which applications are available for customers we've extended the platform with two custom resources. First resource is the [`AppCatalog`]({{< relref "/vintage/use-the-api/management-api/crd/appcatalogs.application.giantswarm.io/" >}}) which is a recipient to collect application definitions that are available to install in the workload clusters. The second is the [`AppCatalogEntry`]({{< relref "/vintage/use-the-api/management-api/crd/appcatalogentries.application.giantswarm.io.md" >}}) which is the representation of the application definition which has a version defined.
```

Comment:
But we dont want to promote happa anymore, it is deprecated and I did want to not mention anymore

Reply:
@AndiDog WDYT?

Reply:
Ok sure, then let's leave out happa and only have the platform API example remaining

Reply:
The above suggestion still holds important diffs, such as `AppCatalog` (old name) vs. `Catalog`, having `$ command` in the first line of the snippets, etc. Please take another look what should be taken over, even if we decided against mentioning happa.

Same below where text snippets should have `$ ` before the command – if and only if there's both command _and_ output in the same snippet. That is inconsistent also in the other sections of the article. We should also use the typical `[...]` continuation format rather than `...`, so it's clear where we cut out the remaining output.

Please try out all snippets since I'm not familiar with querying these things on the command line, and can't easily review which are deprecated CRs and which are new ones.

Reply:
ok I have tried my best, and I also fixed other getting started pages to be consistent

For happa, we leave it out in new docs. We want to add Backstage as soon as it is available to all our customers (current quarter)